app_basics/commands/util.py

```diff
@@ -0,0 +1,42 @@
+def file_tail_or_head(f_name: str, mode: int, n_lines: int = 3):
+    """
+    Function to read first or last n_lines lines from a file
+    f_name - the file relative path
+    mode - 1 is head, -1 is tail
+    n_lines - Number of lines to read
+    """
+
+    with open(f_name, "r") as file:
+        if mode == 1:
```

Comment:
it is better to create constants for the modes.
`MODE_TAIL`, `MODE_HEAD`, etc..

Reply:
it seems that there is too much in common between `file_tail_or_head` and `csv_tail_or_head`.

Reply:
Yes, they are very similar but differ 🤪
In OOP I could have created one abstract class with abstract 'tail_or_head' 
and will have two different implementation one for file and one for CSV. have